CONTRIBUTING.md

```diff
@@ -161,7 +161,7 @@ You can find more information about RIOT development procedure on this
 
 [existing-pull-requests]: https://github.com/RIOT-OS/RIOT/pulls
 [archived-pull-requests]: https://github.com/RIOT-OS/RIOT/pulls?q=is:pr+label:"State:+archived"
-[coding-conventions]: https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions
+[coding-conventions]: https://github.com/RIOT-OS/RIOT/blob/master/CODING_CONVENTIONS.md
```

Comment:
```suggestion
[coding-conventions]: CODING_CONVENTIONS.md
```

should also work and makes it clearer for an off-line reader, that the document is in the checked-out tree.